### DISCUSSION TITLE
Simple Commerce Stripe Integration - passing the postcode

### DISCUSSION THREAD

**Discussion by jamie-b5** [duncanmcclean/simple-commerce#862] [Help] [answered]
Hi, does anyone know how to pass the postcode (from the order data) to the Stripe Element that is created? Or do you pass it in the closure in the Stripe gateway config?

I used to pass it in (in old non Statamic code)  via javascript var card = elements.create('card', {style: style, hidePostalCode: true, value: {postalCode: $postalCode}});

But I am usure as to how to set the $postalCode variable to the current order's shipping or billing post code - do I use some {{sc:order:postal_code}} type antlers syntax?

Thanks! Jamie

**Comment by jamie-b5** [comment, marked as answer]
Answering my own question in case it helps anyone else!

```javascript
 var $postalCode = '{{ billing_postal_code ? billing_postal_code : shipping_postal_code }}';

//const card = elements.create('card');
const card = elements.create('card', {hidePostalCode: true, value: {postalCode: $postalCode}});
```

**Reply by duncanmcclean** [reply]
Glad to hear you managed to find a solution!
